ComAmazonawsDynamodb/codebuild/release-python/validate.yml

```diff
@@ -0,0 +1,46 @@
+version: 0.2
+
+# Validate that the released artifact is usable
+# by installing it from PyPI
+# then running tests from source against the published artifact.
+
+phases:
+  install:
+    commands:
+      - pip install "tox < 4.0"
+    runtime-versions:
+      python: latest
+  pre_build:
+    commands:
+      # Get Dafny
+      - export dafnyVersion=$(grep '^dafnyVersion=' project.properties | cut -d '=' -f 2)
+      - cd ..
+      - curl https://github.com/dafny-lang/dafny/releases/download/v$dafnyVersion/dafny-$dafnyVersion-x64-ubuntu-20.04.zip  -L -o dafny.zip
+      - unzip -qq dafny.zip && rm dafny.zip
+      - export PATH="$PWD/dafny:$PATH"
+      # Switch back to the main directory
+      - cd aws-cryptographic-material-providers-library
+      # Install test dependencies
+      - pyenv install --skip-existing 3.11
+      - pyenv local 3.11
+      - pip install "tox < 4.0"
+      - pip install pytest
+      # Install the published artifact from PyPI.
+      # The tests will use the installed package, not the local code.
+      # The actual test code must be from local, since we don't publish tests.
+      - pip install aws-cryptography-internal-dynamodb==$VERSION
+      # Get CI Creds to be able to call KMS
+      - TMP_ROLE=$(aws sts assume-role --role-arn "arn:aws:iam::370957321024:role/GitHub-CI-MPL-Dafny-Role-us-west-2" --role-session-name "MPL-Python-Test-Release")
```

Comment:
correctness nit: this file is shared by both test and prod releases, so it's incorrect for the latter

Reply:
Yeah... I'll change it to `MPL-Python-Release`